features/tabelwaarden.feature

```diff
@@ -0,0 +1,90 @@
+
+
+  Voor de enumeratie AanduidingBijzonderNederlanderschap wordt de tabel met tabelidentificatie "aanduidingbijzondernederlanderschap" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon behandeld wordt als Nederlander, of dat door de rechter is vastgesteld dat de persoon niet de Nederlandse nationaliteit bezit."
+  In de tabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | B                                                | Behandeld als nederlander        |
+    | V                                                | Vastgesteld Niet-Nederlander     |
+
+  Voor de enumeratie AanduidingBijHuisnummer wordt de tabel met identificatie "aanduidingbijhuisnummer" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "De aanduiding die wordt gebruikt voor adressen die geen straatnaam en huisnummeraanduidingen hebben."
+  In de waardetabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | to                                               | tegenover                        |
+    | by                                               | bij                              |
+
+  Voor de enumeratie Geslacht wordt de tabel met identificatie "geslacht" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan dat de persoon een man of vrouw is, of dat het geslacht (nog) onbekend is."
+  In de waardetabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | M                                                | man                              |
+    | V                                                | vrouw                            |
+    | O                                                | onbekend                         |
+
+  Voor de enumeratie IndicatieGezagMinderjarige wordt de tabel met identificatie "indicatiegezagminderjarige" opgenomen.
+  In de omschrijving wordt de volgende tekst opgenomen : "Geeft aan wie het gezag heeft over de minderjarige persoon."
+  In de waardetabel worden de volgende waarden opgenomen:
+    | code                                             | omschrijving                     |
+    | ------------------------------------------------ | -------------------------------- |
+    | 1                                                | ouder1                           |
```

Comment:
dit is al een bestaande tabel, tabel 61 Gezagsverhoudingtabel: https://publicaties.rvig.nl/dsresource?objectid=223c54d7-433e-4c6d-a68f-0b18fe985f79&type=pdf

Reply:
Beschrijving verwijderd